ocrd/ocrd/workspace.py

```diff
@@ -278,6 +278,7 @@ def _resolve_image_as_pil(self, image_url, coords=None):
 
         with pushd_popd(self.directory):
             pil_image = Image.open(image_filename)
+            pil_image.load() # alloc and give up the FD
```

Comment:
Good that you added a comment, it's not obvious [that `Image.load` closes the FD](https://pillow.readthedocs.io/en/stable/reference/open_files.html#file-handling)